docs_user/modules/proc_migrating-mgr-from-controller-nodes.adoc

```diff
@@ -55,20 +78,21 @@ placement:
   label: mgr
 ----
 
-. Save the spec in the `/tmp/mgr.yaml` file.
+. Save the spec.
 . Apply the spec with `cephadm`` by using the Ceph Orchestrator:
```

Comment:
```suggestion
. Apply the spec with `cephadm` by using the Ceph Orchestrator:
```

Reply:
This wasn't rendering correctly in the preview.

Reply:
Good catch!

Reply:
Done